### ISSUE TITLE
What is the definition of "System level trust"

### ISSUE THREAD

**Issue by Healthedata1** [argonautproject/scheduling#51]
We discussed this concept in the context of provider scheduling across systems.  Is this concept already covered in FHIR or Argonaut?   like here: http://hl7.org/fhir/security.html#http or here: http://hl7.org/fhir/us/core/security.html

I thnk its actually is closer to **[Argonaut Use Case 5](http://argonautwiki.hl7.org/images/4/4c/Argonaut_UseCasesV1.pdf)**  ("Provider	using	EHR	A	access	patient	record	held	in	EHR	B Introduction In	this	use	case,	the	clinician	uses	an	EHR	application	(EHR	A)"

Bottom line is we will need to say a little more about this before publishing the guide.

**Comment by Healthedata1** [comment]
response from cooper:   "System level trust really just means that you are using something like OAuth2 client credentials, or a service account."

look at SMART backend services...
